src/Centreon/Domain/PlatformTopology/Platform.php

```diff
@@ -340,10 +340,6 @@ public function getRelation(): ?PlatformRelation
      */
     public function setRelation(?string $relationType): self
     {
-        if (!in_array($relationType, self::AVAILABLE_RELATIONS)) {
```

Comment:
Shouldn't this check be added in the relation's setter entity ?
https://github.com/centreon/centreon/blob/eb7172b987df2e942931007a2f93b6c67ab0ce24/src/Centreon/Domain/PlatformTopology/PlatformRelation.php#L93